ui/app/components/job-versions-stream.js

```diff
@@ -19,7 +19,9 @@ export default class JobVersionsStream extends Component {
   // Passes through to the job-diff component
   verbose = true;
 
-  @computed('versions.[]')
+  diffs = [];
+
+  @computed('versions.[]', 'diffs.[]')
```

Comment:
the `@computed` prefix to this getter indicates something that should trigger an update.